frontend/public/features.ts

```diff
@@ -82,27 +90,30 @@ const detectOpenShift = dispatch => coFetchJSON(openshiftPath)
       : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift)
   );
 
-const detectCanListNS = dispatch => {
-  const req = {
-    spec: {
-      resourceAttributes: {
-        name: 'namespaces',
-        verb: 'list',
-      },
-    },
-  };
-  k8sCreate(SelfSubjectAccessReviewModel, req)
-    .then(
-      res => setFlag(dispatch, FLAGS.CAN_LIST_NS, res.status.allowed),
-      err => handleError(err, FLAGS.CAN_LIST_NS, dispatch, detectCanListNS)
-    );
+const detectSSARRestrictions = dispatch => {
+  [
+    [ FLAGS.CAN_LIST_NS,    { spec: { resourceAttributes: { name: 'namespaces', verb: 'list' } } }],                // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_NODE,  { spec: { resourceAttributes: { name: 'nodes', verb: 'list' } } }],                     // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_PVC,   { spec: { resourceAttributes: { name: 'persistentvolumeclaims', verb: 'list' } } }],    // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_STORE, { spec: { resourceAttributes: { name: 'storageclasses', verb: 'list' } } }],            // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_CRD,   { spec: { resourceAttributes: { name: 'customresourcedefinitions', verb: 'list' } } }], // eslint-disable-line no-multi-spaces
```

Comment:
I deleted my original comment, but it looks like this should be `resource`, not `name`. (This was originally my bug, although it looks like the k8s docs for selfsubjectaccessreviews are wrong.)

```
$ oc auth can-i list namespaces --loglevel=10
[...]
I0606 16:08:58.485173   19462 round_trippers.go:405] POST https://api.ci.openshift.org:443/apis/authorization.k8s.io/v1/selfsubjectaccessreviews 201 Created in 375 milliseconds
I0606 16:08:58.485213   19462 round_trippers.go:411] Response Headers:
I0606 16:08:58.485221   19462 round_trippers.go:414]     Cache-Control: no-store
I0606 16:08:58.485228   19462 round_trippers.go:414]     Content-Type: application/json
I0606 16:08:58.485236   19462 round_trippers.go:414]     Content-Length: 282
I0606 16:08:58.485243   19462 round_trippers.go:414]     Date: Wed, 06 Jun 2018 20:08:58 GMT
I0606 16:08:58.485299   19462 request.go:897] Response Body: {"kind":"SelfSubjectAccessReview","apiVersion":"authorization.k8s.io/v1","metadata":{"creationTimestamp":null},"spec":{"resourceAttributes":{"namespace":"tectonic-console","verb":"list","resource":"namespaces"}},"status":{"allowed":true,"reason":"allowed by openshift authorizer"}}
yes
```

Can you test with cluster-reader instead of cluster-admin?

```
$ oc adm add-cluster-role-to-user cluster-reader <user-id>
```

Reply:
Do we need `eslint-disable-line no-multi-spaces`? I'd rather just conform to what the linter wants, and then the code remains consistent.

Reply:
I can do that, I just liked the uniformity :)
I'll remove.